dotCMS/src/main/java/com/dotcms/contenttype/model/field/KeyValueField.java

```diff
@@ -46,7 +44,7 @@ public DataTypes dataType(){
 	@JsonIgnore
 	public Collection<ContentTypeFieldProperties> getFieldContentTypeProperties(){
 		return list(ContentTypeFieldProperties.NAME, ContentTypeFieldProperties.REQUIRED, ContentTypeFieldProperties.HINT,
-				ContentTypeFieldProperties.SEARCHABLE);
+				ContentTypeFieldProperties.SEARCHABLE, ContentTypeFieldProperties.INDEXED);
```

Comment:
any test?

Reply:
What would you test there? This is a change we're rolling back and re-introducing just this bit.

Reply:
this is rollback of something already merged